src/psyclone/domain/nemo/transformations/nemo_allarrayrange2loop_trans.py

```diff
@@ -111,8 +116,16 @@ def apply(self, node, options=None):
         try:
             while True:
                 trans.apply(node)
-        except TransformationError:
-            pass
+        except TransformationError as err:
+            if options and options.get("verbose", False):
+                errmsg = str(err)
+                # Skip the errors that are obious and are generated for any
+                # statement that is not an array expression
+                if "assignment node should be an expression with an array " \
+                   "that has a Range node" not in errmsg and \
+                   "be a Reference that contains an array access somewhere" \
+                   not in errmsg:
+                    print(errmsg)
```

Comment:
I added the TODO.

I have not changed to stderr because the counterpoint is that in the examples/nemo/scripts all the similar messages are in stdout, and I use some commands to count the instances of each error so I would like that they are all printed to the same stream.

But I do see the point, is this done in other transformation errors? I guess I could still 2>1 for my use case.